Resources/Prototypes/_DV/Entities/Objects/Specific/Chapel/bibles.yml

```diff
@@ -0,0 +1,217 @@
+- type: entity
+  parent: Bible
+  id: Quran
+  suffix: "Bible"
+  name: qur'an
+  description: "Religious text of Space Islam with a decorated cover."
+  components:
+    - type: Sprite
+      sprite: Objects/Specific/Chapel/quran.rsi
+      state: icon
+    - type: Item
+      size: Small
+      sprite: Objects/Specific/Chapel/quran.rsi
```

Comment:
unindent by 2 spaces, dont need to copy paste state and size since they are inherited